_faq/14-app-context-vs-singleton.asciidoc

```diff
@@ -0,0 +1,37 @@
+= What's the difference between `@ApplicationScoped` and `@Singleton`
+
+Both these contexts behave very similarly.
+A single bean instance is created and shared across the application.
+However, there are some important differences.
+
+== Client Proxy
+
+First of all, `@ApplicationScoped` is a *normal* scope whereas `@Singleton` is a *pseudo-scope* (using the CDI terminology).
+What does it mean?
+In the first place, for normal scopes a https://docs.jboss.org/cdi/spec/2.0/cdi-spec.html#client_proxies[client proxy] is always injected.
+This is a container construct that delegates all method calls to the current bean instance.
+Looks like some unnecessary overhead but in fact allows the container to do some useful stuff.
```

Comment:
This sounds a little too vague, how about
`While this may seem as an unnecessary overhead, it allows Weld/CDI to to the following:`